lib/postgrex/replication.ex

```diff
@@ -488,65 +534,52 @@ defmodule Postgrex.Replication do
   end
 
   @doc """
-  Starts streaming a copy of the given table.
-
-  This function can be used to initially synchronize the target system with
-  the primary PostgreSQL instance before starting replication. For increased
-  performance, a pool of replication processes can be created, each one
-  synchronizing a single table at at time.
-
-  The function performs the following steps:
-
-    * A transaction will begin with the `REPEATABLE READ` isolation level.
-
-    * A replication slot will be created with option `:snap_shot = :use`.
-      This ensures the data read during the transaction is consistent with
-      the way it  was when the slot was created.
-
-    * A `COPY` command will be issued for the given table and the rows
-      will be streamed to `handle_data/2`. The messages will be of the type
-      `Postgrex.Result.t()` with the row values having `text` format.
-
-    * Once all the rows have been copied, a final message of the form
-      `{:copy_done, table_name}` will be sent, the transaction will be
-      committed and the slot (optionally) dropped.
-
-  This function will return the created slot's information, the same as
-  `create_slot/4`. This information may be useful, for example, with the
-  following tasks:
-
-    * Using the slot's `consistent_point` to determine where to start
-      replication.
-
-    * Using the slot to catch up with the changes that occurred during copying.
-      In this scenario, there may be a single main replication process alongside
-      secondary replication processes for each table. An individual secondary process
-      can be used to copy the initial snapshot for a given table and then stream
-      replication messages for that table until it is caught up to the main process.
-      Once caught up, the secondary process can be terminated and control of the table
-      given back to the main process. This strategy minimizes the amount of time a
-      transaction must hold onto old transaction IDs as well as the amount of time
-      a slot must hold onto old WAL files.
-
-  If the connection was started with `auto_reconnect` set to `true`, then
+  Streams a copy of the given table.
+
+  This function is meant to aid in the initial synchronization of large
+  databases with high traffic. For simpler situations, it may be desirable
+  to use a non-replication connection to copy the table.
+
+  In addition to copying the table, a replication slot will be created using
+  the `slot_name`, `plugin` and `:temporary` values passed into this function.
+  The replication slot is returned to the user so it can be used to catch up
+  with the changes that occur during copying. The slot's `:snapshot` value
+  is automatically set to `:use` to ensure the user can start replication where
+  the copying transaction left off.
```

Comment:
```suggestion
  In addition to copying the table, a replication slot will be created using
  the `slot_name` and `plugin` arguments passed into this function. The slot
  is temporary by default but it can be changed via the `:temporary` option.
  The replication slot is returned to the user so it can be used to catch up
  with the changes that occur during copying. The slot's `:snapshot` value
  is automatically set to `:use` to ensure the user can start replication where
  the copying transaction left off.
```